### PR TITLE
Move post deployment action to the client

### PR DESCRIPTION
**Pull request by jmcs**
See also zalando/lizzy#108 and zalando/lizzy#118.

This PR also includes some changes to make lizzy-client's output closer to senza's.

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -44,37 +45,45 @@ def new_stack(self, image_version, keep_stacks, traffic, definition,
...
                     'creation_time': '2016-01-01T12:00:00Z'}
 
-    def traffic(self, stack_id, percentage):
-        ...
-
     def wait_for_deployment(self, stack_id: str) -> [str]:
         return ['CF:WAITING', self.final_state]
 
-    def get_stacks(self) -> list:
+    def get_stacks(self, stack_reference=None) -> list:
```
Comment (rafaelcaricio) [inline tests/test_cli.py]: I think would be better to mock the payloads of HTTP call to lizzy agent than mocking the whole logic here. What do you think?

Comment (rafaelcaricio) [inline tests/test_cli.py]: Looks like the tests are testing the mocks.

**Comment by rafaelcaricio** [comment]
👍

File: tests/test_cli.py (hunk 1 above)
Comment (jmcs) [inline tests/test_cli.py]: I improved the tests a little bit.

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**